video2dataset/workers/subset_worker.py

```diff
@@ -67,54 +118,102 @@ def process_shard(
         failed_to_subsample = 0
         error_message = None
 
+        if "s3" in shard:
+            shard = f"pipe:aws s3 cp {shard} -"
```

Comment:
I don't think we want this... for every project you just add the pipe... I think there's a reason for that

Reply:
I think because like this assumes you're using aws s3